### ISSUE TITLE
[A11y] [Issue 5] Extra focusable element that adds confusion to the interaction within chat

### ISSUE THREAD

**Issue by compulim** [microsoft/BotFramework-WebChat#3140]
# 5 Severity 2 – Extra focusable element that adds confusion to the interaction within chat

> This is filed to https://github.com/microsoft/BotFramework-WebChat/issues/3140.

> This is related to Adaptive Cards.

## Background

Unnecessary and confusing Tab stop around “I agree” and “I don’t agree” buttons, as well as the age, country and state controls. For I agree/don’t agree, the screen reader announces “Frame clickable main landmark complimentary landmark list with one item I agree button I don’t agree” It is not clear to the users what action can be taken here. Pressing Enter or Space while focused on that element does nothing. Tab Stop should be removed from this element. It is especially burdensome in the below dialog, as it announces all the options at once

## Investigation notes

This is a bug on Adaptive Cards renderer that emitted a `tabindex="0"` even for non-clickable cards. By clickable, I mean, the card frame can be clickable, regardless whether the card contains any buttons or not.

Most cards are not clickable, but they still have the `tabindex="0"` on it.

[Bug]

**Comment by corinagum** [comment]
Closing this as resolved.
